auth-web/src/components/auth/staff/continuation-application/AuthorizationContactInformation.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div
+    v-if="!!review && !!filing"
+    id="authorization-contact-information"
+  >
+    <!-- Authorization Contact Information -->
+    <section class="section-container">
+      <v-row no-gutters>
+        <!-- First Column: Contact Information Label -->
+        <v-col
+          cols="12"
+          sm="4"
+        >
+          <label>Contact Information</label>
+        </v-col>
+        <!-- Second Column: Email Address Label and Value -->
+        <v-col
+          cols="12"
+          sm="4"
+        >
+          <label>Email Address</label>
+          <div>{{ email || '[Unknown]' }}</div>
+        </v-col>
```

Comment:
There's something indenting the email address. It should align with the data in the sections above and below (see design). Please fix.

![image](https://github.com/user-attachments/assets/63fb4d4d-c840-4564-b88c-ebe2c17bf6db)

Reply:
Updated:
![image](https://github.com/user-attachments/assets/770a96dd-5624-48d8-a10f-916691e109e1)